templates/_bundle_builder/Dockerfile

```diff
@@ -0,0 +1,89 @@
+FROM --platform=${TARGETPLATFORM} liferay/base:latest
+
+RUN mkdir /build
+
+# Common
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes build-essential ca-certificates curl libfreetype6-dev libjpeg-dev liblzma-dev libpng-dev libtiff-dev pkg-config zlib1g-dev
+
+# Ffmpeg
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --yes libass-dev libfaac-dev libmp3lame-dev libopus-dev libsdl2-dev libtheora-dev libvorbis-dev libvpx-dev libx264-dev libx265-dev yasm
+
+RUN curl -L -o /build/ffmpeg.tar.gz https://ffmpeg.org/releases/ffmpeg-7.1.tar.gz && \
```

Comment:
@Akos-Kreutz, can you please use the full name of the parameters, like `--location` and `--output`, instead?

BChan prefers it since they provide more clearness of their purpose. :slightly_smiling_face: 

Sorry for miss it in the first review. :grin: 

Can you please apply it across all this file?

Thanks in advance